pom.xml

```diff
@@ -34,80 +34,92 @@
         <dependency>
             <groupId>org.apache.derby</groupId>
             <artifactId>derby</artifactId>
-            <version>10.12.1.1</version>
+            <version>10.13.1.1</version>
         </dependency>
-             <dependency>
-    <groupId>javax.json</groupId>
-    <artifactId>javax.json-api</artifactId>
-    <version>1.0</version>
-</dependency>
-    <dependency>
-        	<groupId>javax.servlet</groupId>
-        	<artifactId>javax.servlet-api</artifactId>
-        	<scope>provided</scope>
-        	<version>3.1.0</version>
+        <dependency>
+            <groupId>javax.json</groupId>
+            <artifactId>javax.json-api</artifactId>
+            <version>1.0</version>
         </dependency>
- 
- <dependency>
-	<groupId>javax.transaction</groupId>
-	<artifactId>javax.transaction-api</artifactId>
-	<version>1.2-b03</version>
-</dependency>
-
-         <dependency>
+        <dependency>
+            <groupId>javax.servlet</groupId>
+            <artifactId>javax.servlet-api</artifactId>
+            <scope>provided</scope>
+            <version>3.1.0</version>
+        </dependency>
+        <dependency>
+            <groupId>javax.transaction</groupId>
+            <artifactId>javax.transaction-api</artifactId>
+            <version>1.2-b03</version>
```

Comment:
This is a problem in the original sample.  javax.transaction-api dependency should have `provided` scope.